lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/SqlQuery.java

```diff
@@ -178,6 +182,20 @@ public SqlQuery(
 			SelectList expandedSelectList = SQLTranslatorUtils.createSelectListFromSchema(tableSchema);
 			this.model.replaceSelectList(expandedSelectList);
 		}
+
+
+		//Append additionalFilters onto the WHERE clause
+		if(additionalFilters != null && !additionalFilters.isEmpty()) {
+			String additionalFilterSearchCondition = SQLUtils.appendAdditionalFilters(additionalFilters);
+			StringBuilder whereClauseBuilder = new StringBuilder();
+			FacetUtils.appendFacetWhereClauseToStringBuilderIfNecessary(whereClauseBuilder,additionalFilterSearchCondition, this.model.getTableExpression().getWhereClause());

```

Comment:
We could probably make this method generic (maybe move it to some query/sql util class) and reuse it for facets